src/plugins/data_views/server/content_management/data_views_storage.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SOContentStorage } from '@kbn/content-management-utils';
+
+import type { DataViewCrudTypes } from '../../common/content_management';
+import { DataViewSOType } from '../../common/content_management';
+import { cmServicesDefinition } from '../../common/content_management/cm_services';
+
+export class DataViewsStorage extends SOContentStorage<DataViewCrudTypes> {
```

Comment:
No, its the service definition. The service definition is passed in to `SOContentStorage` so it can be applied in a consistent manner for different SO types. Since we only have v1, there's nothing to code for yet but there's an example here - `src/plugins/content_management/docs/content_onboarding.md:587`